src/initializer.tsx

```diff
@@ -0,0 +1,146 @@
+/*
+Copyright 2021 New Vector Ltd
```

Comment:
matrix.org copyright, also the right year :)

Reply:
I thought since this is in `vector-im` we should use `New Vector LTD`?

Reply:
I updated the year and matrix.org.
It seems we are using both in element call.
Should we unify that and move everything to one of the two?